filesystem/templatecopy.go

```diff
@@ -88,7 +88,7 @@ func templateCopyAdditionCallBack(source, destination string, config interface{}
 func templateCopyDeletionCallBack(source, destination string, config interface{}) error {
 	si, err := os.Stat(source)
 	if err != nil {
-		logrus.Errorf("Unable to stat %s : %s", source, err)
+		logrus.Errorf("Unable to stat %s in templateCopyDeletionCallBack(): %s", source, err)
```

Comment:
Since this is a error message let's not expose function names here. Probably change it wo 'Unable to stat [%s] while processing deletion during template copy'

Reply:
done